elections/steering/2024/nomination-template.md

```diff
@@ -0,0 +1,23 @@
+-------------------------------------------------------------
+name: 
+ID: GitHubID
+info:
+  - employer: Your Employer or "Independent"
+  - slack: slack handle
+-------------------------------------------------------------
+
+<!-- Please make a copy of this template as "candidate-githubid.md" and save it to
+the election directory -->
+
+## SIGS
+
+- SIGS/WG/UGs you're a member of
```

Comment:
Does UGs in this context refer to the User Groups I am removing a reference to? Or do we have it here to be more broadly applicable?

Reply:
Yes, that's another stale reference to user groups. We should drop `/UG`

Reply:
... Also personally I would nitpick SIGS => SIGs (since the s is plurality, not the acronym) but I don't think think that's important.

Reply:
I'd change it to:

`SIGs/WGs/Teams you're a member of`

Reply:
Done (which removed the LGTM).